metadata_service/dao/study.py

```diff
@@ -87,7 +98,11 @@ async def update_study(study_id: str, data: Dict) -> Dict:
       The updated Study object
 
     """
-    collection = await get_collection(COLLECTION_NAME)
-    await collection.update_one({"id": study_id}, {"$set": data})
+    db_connect = DBConnect()
+    collection = await db_connect.get_collection(COLLECTION_NAME)
+    await collection.update_one(  # type: ignore
+        {"id": study_id}, {"$set": data.dict(exclude_unset=True)}
+    )
     study = await get_study(study_id)
+    await db_connect.close_db()
```

Comment:
This is aggressively closing the db connection. The connection is closed by `get_study` in the line above.

Reply:
fixed it